projects/angular/src/data/tree-view/tree-node-link.ts

```diff
@@ -10,7 +10,11 @@ import { Directive, ElementRef } from '@angular/core';
   selector: '.clr-treenode-link',
 })
 export class ClrTreeNodeLink {
-  constructor(private el: ElementRef) {}
+  constructor(private el: ElementRef<HTMLElement>) {}
+
+  get active() {
+    return this.el.nativeElement.classList.contains('active');
```

Comment:
I couldn't seem to get `@HostBinding('class.active') active: boolean;` to work for this. I chalked it up to ng-clarity weirdness, but any suggestions are welcome.